src/helpers/utils.js

```diff
@@ -32,7 +32,7 @@ export default {
    * @returns {string} - return jwt token
    */
   jwtSigner(payload) {
-    return jwt.sign(payload, process.env.JWT_SECRET, { expiresIn: '24h' });
+    return jwt.sign(payload, process.env.JWT_SECRET);
```

Comment:
Please return the expiry time or make it an variable that can be configured on `process.env`. Tokens are meant to expire